tests/integration/cli/tests/create_exe.rs

```diff
@@ -483,7 +483,8 @@ fn create_exe_works_with_file() -> anyhow::Result<()> {
 
 // Ignored because of -lunwind linker issue on Windows
 // see https://github.com/wasmerio/wasmer/issues/3459
-#[cfg_attr(target_os = "windows", ignore)]
```

Comment:
Please add `TODO: ` comments here, and also create an issue (assigned to Felix) to fix this, otherwise we will definitely forget.

Reply:
To clarify: TODO on both tests.

Reply:
Done